README.md

```diff
@@ -0,0 +1,8 @@
+# KNI debug tools
+
+The purpose of this repo is to host debug, troubleshooting, test tools used to troubleshoot low-latency KNI cluster, pod, workload configurations.
+We aim to distill and consolidate the debug/troubleshooting flows in a set of tools.
+
+In order to be composable, the logic must be pushed into reusable packages, from where we can build command line utilities, that
+we can import in other projects [or operators](https://github.com/openshift-kni/performance-addon-operators/), or that we can pack on automated
+troubleshoot helpers.
```

Comment:
I would suggest a short description on how it is advised to be consumed. As a container or some other way.

Reply:
Good idea. I will add in the next upload.